### DISCUSSION TITLE
Does this dynamic effect make your eyes a little uncomfortable?

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1401] [General]
Each time close a dialog, the page always slides，is it necessary?
![GIF 2023-08-12 11-23-28](https://github.com/zauberzeug/nicegui/assets/25096045/382c418b-ffc2-4392-8cfb-62f987e7337b)

**Comment by rodja** [comment]
Wow! Thanks for reporting @v479038280; this animation is not intended. Definitely a bug.

**Comment by rodja** [comment]
Here is the issue: https://github.com/zauberzeug/nicegui/issues/1404
Let's do all further discussions there.
